src/lib/services/digital-asset.service.ts

```diff
@@ -157,7 +164,9 @@ function initializeLSP7Proxy(
   digitalAssetDeploymentReceipt$: Observable<DeploymentEventProxyContract>,
   digitalAssetDeploymentOptions: LSP7DigitalAssetDeploymentOptions
 ) {
-  const { name, symbol, isNFT } = digitalAssetDeploymentOptions;
+  const { name, symbol, tokenType, isNFT } = digitalAssetDeploymentOptions;
+
+  const lsp4TokenType = typeof tokenType === 'string' ? LSP4_TOKEN_TYPES[tokenType] : tokenType;
```

Comment:
Same here I think it is safer to verify this value.

Reply:
same reply as above